examples/guestbook/README.md

```diff
@@ -260,59 +303,49 @@ redis-master            name=redis-master                         name=redis-mas
 redis-slave             name=redis-slave                          name=redis-slave             10.0.72.62          6379
 ```
 
-### Step Five: Create the frontend pod
-
-This is a simple PHP server that is configured to talk to either the slave or master services depending on whether the request is a read or a write. It exposes a simple AJAX interface, and serves an angular-based UX. Like the redis read slaves it is a replicated service instantiated by a replication controller.
-
-It can now leverage writes to the load balancing redis-slaves, which can be highly replicated.
-
-The pod is described in the file `examples/guestbook/frontend-controller.json`:
-
-```js
-{
-   "kind":"ReplicationController",
-   "apiVersion":"v1",
-   "metadata":{
-      "name":"frontend",
-      "labels":{
-         "name":"frontend"
-      }
-   },
-   "spec":{
-      "replicas":3,
-      "selector":{
-         "name":"frontend"
-      },
-      "template":{
-         "metadata":{
-            "labels":{
-               "name":"frontend"
-            }
-         },
-         "spec":{
-            "containers":[
-               {
-                  "name":"php-redis",
-                  "image":"kubernetes/example-guestbook-php-redis:v2",
-                  "ports":[
-                     {
-                        "containerPort":80
-                     }
-                  ]
-               }
-            ]
-         }
-      }
-   }
-}
+### Step Five: Create the frontend replicated pods
+<a href="#step-five-create-the-frontend-replicated-pods"></a>
+
+A frontend pod is a simple PHP server that is configured to talk to either the slave or master services, depending on whether the client request is a read or a write. It exposes a simple AJAX interface, and serves an Angular-based UX.
+Again we'll create a set of replicated frontend pods instantiated by a replication controller— this time, with three replicas.
+
+The replicas of the frontend pod can leverage writes to the load-balancing redis-slaves, which can be highly replicated as well.
```

Comment:
hmm, I didn't write that either :).  I removed that line and added a line below the PHP script listing that I think gets at the gist of what was intended.